src/Auth/authSaga.js

```diff
@@ -0,0 +1,8 @@
+import { select, take } from 'redux-saga/effects';
+import { LOGIN_REQUEST } from './authModule';
+
+export default function* loginFlow() {
+  yield take(LOGIN_REQUEST);
+  const { authModule } = yield select();
+  localStorage.setItem('KIZUNA_USERNAME', authModule.username);
```

Comment:
I'll probably try to find a middleware to sync the redux store and localStorage...